.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+jobs:
+  bump-version:
+    if: github.event.pull_request.merged == true
```

Comment:
This is necessary to ensure that the workflow only runs when PRs are merged not when they are closed without merging